packages/config/packages/config/config/devices/0x0371/zw141.json

```diff
@@ -0,0 +1,503 @@
+{
+	"manufacturer": "Aeotec Limited",
+	"manufacturerId": "0x0371",
+	"label": "ZW141",
+	"description": "Nano Shutter",
+	"devices": [
+		{
+			"productType": "0x0003",
+			"productId": "0x008d"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Retransmit",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": {
+		"20": {
+			"label": "Motor behavior after powering on",
+			"description": "Set the motor behavior after powering on",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 0,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Stay the same position",
+					"value": 0
+				},
+				{
+					"label": "Move to 99%",
+					"value": 1
+				},
+				{
+					"label": "Move to 0%",
+					"value": 2
+				}
+			]
+		},
+		"22": {
+			"label": "Motor direction",
+			"description": "Set to toggle the motor running direction",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 2,
+			"unsigned": true,
+			"readOnly": false,
+			"writeOnly": false,
+			"allowManualEntry": false
+		},
+		"34": {
+			"label": "Blade turn time",
+			"description": "Set the blade turn time for Venetian mode. Details can be found in section 4.2 of Advanced information Product Manual.",
+			"valueSize": 2,
+			"unit": "seconds",
+			"minValue": 10,
+			"maxValue": 32767,
```

Comment:
This maxValue suggests the unit isn't actually seconds. It must be ms or something. Please verify and change.